src/app/shared/entry/doi/manage-dois/manage-dois-dialog.component.html

```diff
@@ -87,6 +87,42 @@ <h1 mat-dialog-title>Manage DOIs</h1>
                 <app-doi-badge *ngIf="doiInfo.value.conceptDoi" [doi]="doiInfo.value.conceptDoi" [displayInitiator]="false"></app-doi-badge>
               </span>
             </div>
+
+            <div class="mt-3" *ngIf="doiInfo.key === DoiInitiatorEnum.DOCKSTORE">
+              <div class="size-small gray-caption">
+                You can edit the metadata of a published Dockstore DOI at anytime.
+                <button mat-button (click)="toggleShowMoreEditLinkInfo()" class="private-btn">
+                  <mat-icon class="mat-icon-sm">{{ showMoreEditLinkInfo ? 'keyboard_arrow_up' : 'keyboard_arrow_down' }}</mat-icon
+                  >{{ showMoreEditLinkInfo ? 'Collapse' : 'Show more' }}
+                </button>
+              </div>
+              <div *ngIf="showMoreEditLinkInfo">
+                <p class="size-small gray-caption">
+                  Dockstore uses Zenodo's link sharing feature to create a secret link with edit access so you can edit the record. You must
+                  be logged into your Zenodo account when you open the link. When you are done editing the record, delete the link on
```

Comment:
Maybe "your Zenodo account" should "a Zenodo account" on the off chance that the account they link in the accounts page is different. Minor point